assets/js/components/OverlayNotification/AnalyticsAndAdSenseAccountsDetectedAsLinkedOverlayNotification.js

```diff
@@ -0,0 +1,236 @@
+		)
+	);
+
+	const { setOverlayNotificationToShow, dismissOverlayNotification } =
+		useDispatch( CORE_UI );
+
+	const isDismissed = useSelect( ( select ) =>
+		select( CORE_USER ).isItemDismissed(
+			ANALYTICS_ADSENSE_LINKED_OVERLAY_NOTIFICATION
+		)
+	);
+
+	const isDismissing = useSelect( ( select ) =>
+		select( CORE_USER ).isDismissingItem(
+			ANALYTICS_ADSENSE_LINKED_OVERLAY_NOTIFICATION
+		)
+	);
+
+	const analyticsModuleConnected = useSelect( ( select ) => {
+		if ( isDismissed ) {
+			return null;
+		}
+		return select( CORE_MODULES ).isModuleConnected( 'analytics-4' );
+	} );
+
+	const adSenseModuleConnected = useSelect( ( select ) => {
+		if ( isDismissed ) {
+			return null;
+		}
+		return select( CORE_MODULES ).isModuleConnected( 'adsense' );
+	} );
+
+	const canViewSharedAnalytics = useSelect( ( select ) => {
+		if ( isDismissed ) {
+			return null;
+		}
+		return select( CORE_USER ).hasAccessToShareableModule( 'analytics-4' );
+	} );
+	const canViewSharedAdSense = useSelect( ( select ) => {
+		if ( isDismissed ) {
+			return null;
+		}
+		return select( CORE_USER ).hasAccessToShareableModule( 'adsense' );
+	} );
+
+	const isAdSenseLinked = useSelect( ( select ) => {
+		if ( isDismissed ) {
+			return null;
+		}
+
+		return select( MODULES_ANALYTICS_4 ).getAdSenseLinked();
+	} );
+
+	const adSenseAccountID = useSelect( ( select ) => {
+		if ( adSenseModuleConnected ) {
+			return select( MODULES_ADSENSE ).getAccountID();
+		}
+
+		return null;
+	} );
+
+	const { startDate, endDate } = useSelect( ( select ) =>
+		select( CORE_USER ).getDateRangeDates( {
+			offsetDays: DATE_RANGE_OFFSET,
+		} )
+	);
+
+	const args = {
+		startDate,
+		endDate,
+		dimensions: [ 'pagePath', 'adSourceName' ],
+		metrics: [ { name: 'totalAdRevenue' } ],
+		filter: {
+			fieldName: 'adSourceName',
+			stringFilter: {
+				matchType: 'EXACT',
+				value: `Google AdSense account (${ adSenseAccountID })`,
+			},
+		},
+		orderby: [ { metric: { metricName: 'totalAdRevenue' }, desc: true } ],
+		limit: 1,
+	};
+
+	const data = useSelect( ( select ) => {
+		if (
+			! isDismissed &&
+			isAdSenseLinked &&
+			adSenseModuleConnected &&
+			analyticsModuleConnected &&
+			canViewSharedAdSense &&
+			canViewSharedAnalytics
+		) {
+			return select( MODULES_ANALYTICS_4 ).getReport( args );
+		}
+
+		return null;
+	} );
+
+	const dataAvailable = !! data?.rows?.length;
+
+	const shouldShowNotification =
+		isDismissed === false &&
+		ga4AdSenseIntegration &&
+		analyticsModuleConnected &&
+		adSenseModuleConnected &&
+		canViewSharedAnalytics &&
+		canViewSharedAdSense &&
+		isAdSenseLinked &&
+		dataAvailable;
+
+	const dismissNotification = useCallback( () => {
+		// Dismiss the notification, which also dismisses it from
+		// the current users profile with the `dismissItem` action.
+		dismissOverlayNotification(
+			ANALYTICS_ADSENSE_LINKED_OVERLAY_NOTIFICATION
+		);
+	}, [ dismissOverlayNotification ] );
+
+	const scrollToWidget = ( event ) => {
```

Comment:
According to the AC this should also dismiss notification, and then scroll to the section